### PR TITLE
VDS-143,VDS-144: Migrate home page to bootstrap4

### PR DESCRIPTION
**Pull request by ilyawzrd**
Снимок экрана 2020-11-26 в 11 26 27
Снимок экрана 2020-11-26 в 11 26 54
Снимок экрана 2020-11-26 в 11 26 32
Снимок экрана 2020-11-26 в 11 26 39
**Index Page**
Снимок экрана 2020-11-26 в 19 42 31
**Header for account page**
Снимок экрана 2020-11-27 в 15 06 24
Снимок экрана 2020-11-27 в 15 06 38

### CODE REVIEW

**Review by trueboroda — APPROVED** [review]

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: assets/scss/bootstrap-migration/_maps.scss [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-  "success":   $success,
-  "info":      $info,
-  "warning":   $warning,
-  "danger":    $danger,
-  "gray-900":  $gray-900
+$theme-colors: map-merge(
+  $theme-colors,
+  (
+    'gray-900': $gray-900,
+    'gray-200': $gray-200,
```
[the remark this answers is not in the document]
Comment (asvishnyakov) [reply assets/scss/bootstrap-migration/_maps.scss:5]: Everything is ok, it's theme colors

File: assets/scss/bootstrap-migration/custom/_navbar.scss [hunk 2]
```diff
@@ -1,4 +1,13 @@
 .navbar-search-bar {
-    min-height: 3.25rem;
-    font-size: $font-size-base;
+  min-height: 3.25rem;
+}
```
Comment (asvishnyakov) [inline assets/scss/bootstrap-migration/custom/_navbar.scss:3]: Replace with flex stretch in template

File: assets/scss/bootstrap-migration/custom/_navbar.scss [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+
+.navbar.navbar-dark {
+  li.active > {
+    a,
+    a:hover {
+      color: $body-color;
```
Comment (asvishnyakov) [inline assets/scss/bootstrap-migration/custom/_navbar.scss:9]: Probably will be replaced with correct navbar template

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
Rename `custom` folder to `override` and move our own custom styles like `bulk-order-popover` to new `custom` folder

File: bower.json [hunk 4]
```diff
@@ -36,7 +36,8 @@
     "polyfills": "*",
     "prism": "^1.8.4",
     "underscore": "^1.8.3",
-    "owl.carousel": "^2.3.4"
+    "owl.carousel": "^2.3.4",
+    "ui-bootstrap4": "morgul/ui-bootstrap4#^3.0.7"
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply bower.json]: done

File: assets/scss/bootstrap-migration/custom/_buttons.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.btn {
+  text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_buttons.scss:2]: done

File: assets/scss/bootstrap-migration/custom/_popover.scss [hunk 6]
```diff
@@ -0,0 +1,58 @@
+.popover {
+  &.popover-square {
+    border-radius: 0;
+
+    > * {
+      border-radius: 0;
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_popover.scss]: done

File: assets/scss/bootstrap-migration/custom/_forms.scss [hunk 7]
```diff
@@ -0,0 +1,36 @@
+.form-control {
+  box-shadow: none;
+  font-size: $font-size-base;
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_forms.scss]: done

File: assets/scss/bootstrap-migration/custom/_forms.scss [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+  &:focus {
+    box-shadow: 0 0 0 1px $input-focus-border-color inset;
+  }
+}
+
+textarea::placeholder {
+  font-size: $font-size-base;
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_forms.scss]: done

File: assets/scss/bootstrap-migration/custom/_dropdowns.scss [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+      a:hover {
+        color: $white;
+      }
+    }
+
+    > a {
+      color: $gray-900;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_dropdowns.scss]: done

File: assets/scss/bootstrap-migration/custom/_navbar.scss (hunk 2 above)
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_navbar.scss]: done

File: assets/scss/bootstrap-migration/custom/_navbar.scss (hunk 3 above)
Comment (ilyawzrd) [reply assets/scss/bootstrap-migration/custom/_navbar.scss]: Navbar template fixed, but this code isn't replaced, because it's needed for active state

**Review by asvishnyakov — DISMISSED** [review]

**Review by asvishnyakov — DISMISSED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**